File: blinker.cpp
// Blinker - random LED patterns
#include "MicroBit.h"

MicroBit uBit;
// MicroBitDisplay display;

int main()
{
    uBit.seedRandom();
    int start = uBit.systemTime();
    for (int i = 0; i < 100000; i++) {
        uBit.display.image.setPixelValue(uBit.random(5), uBit.random(5), uBit.random(256));
        uBit.display.image.setPixelValue(uBit.random(5), uBit.random(5), 0);
    }
    int end = uBit.systemTime();
    while (1) {
        uBit.display.clear();
        uBit.display.scroll(end-start);
        uBit.sleep(1000);
    }
    // 5595 - 5609
}

File: blinker.py
# Blinker - random LED patterns
from microbit import *
import random

start = running_time()
for i in range(100000):
	display.set_pixel(random.randrange(5), random.randrange(5), random.randrange(10))
	display.set_pixel(random.randrange(5), random.randrange(5), 0)
end = running_time()
while True:
    display.clear()
    display.scroll("%d" % (end - start))
    sleep(1000)
# 68232